### PR TITLE
Update Links

### PR DESCRIPTION
**Pull request by mikeamoss**
Fix broken links from legacy MSDN docs to current MS Docs links.

### CODE REVIEW

File: WindowsServerDocs/identity/ad-ds/reference/virtual-dc/Virtualized-Domain-Controller-Cloning-Test-Guidance-for-Application-Vendors.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Additionally, the new computer has a different IP address than the original comp
...
   
-To induce a cloning failure, try to clone a domain controller without granting it permission to be cloned. In this case, the computer will have only changed the IP addresses and still have the majority of its state from the original domain controller. For more information about granting a domain controller permission to be cloned, see [Step 1: Grant the source virtualized domain controller the permission to be cloned](https://technet.microsoft.com/library/hh831734.aspx#bkmk4_grant_source).  
+To induce a cloning failure, try to clone a domain controller without granting it permission to be cloned. In this case, the computer will have only changed the IP addresses and still have the majority of its state from the original domain controller. For more information about granting a domain controller permission to be cloned, see [Step 1: Grant the source virtualized domain controller the permission to be cloned](https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/get-started/virtual-dc/virtualized-domain-controller-deployment-and-configuration).
```
Comment (ItzLevvie) [inline WindowsServerDocs/identity/ad-ds/reference/virtual-dc/Virtualized-Domain-Controller-Cloning-Test-Guidance-for-Application-Vendors.md]:
```suggestion
To induce a cloning failure, try to clone a domain controller without granting it permission to be cloned. In this case, the computer will have only changed the IP addresses and still have the majority of its state from the original domain controller. For more information about granting a domain controller permission to be cloned, see [Step 1: Grant the source virtualized domain controller the permission to be cloned](https://docs.microsoft.com/windows-server/identity/ad-ds/get-started/virtual-dc/virtualized-domain-controller-deployment-and-configuration).  
```

File: WindowsServerDocs/identity/ad-ds/reference/virtual-dc/Virtualized-Domain-Controller-Cloning-Test-Guidance-for-Application-Vendors.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ A domain controller that runs your application or service cannot be cloned until
...
+The first time the user runs the Get-ADDCCloningExcludedApplicationList cmdlet, it returns a list of services and applications that are running on the domain controller but are not in the default list of services and applications that are supported for cloning. By default, your service or application will not be listed. To add your service or application to the list of applications and services that can be safely cloned, the user runs Get-ADDCCloningExcludedApplicationList cmdlet again with the -GenerateXML option in order to add it to the CustomDCCloneAllowList.xml file. For more information, see [Step 2: Run Get-ADDCCloningExcludedApplicationList cmdlet](https://docs.microsoft.com/en-us/powershell/module/addsadministration/get-addccloningexcludedapplicationlist).
```
Comment (ItzLevvie) [inline WindowsServerDocs/identity/ad-ds/reference/virtual-dc/Virtualized-Domain-Controller-Cloning-Test-Guidance-for-Application-Vendors.md]:
```suggestion
The first time the user runs the Get-ADDCCloningExcludedApplicationList cmdlet, it returns a list of services and applications that are running on the domain controller but are not in the default list of services and applications that are supported for cloning. By default, your service or application will not be listed. To add your service or application to the list of applications and services that can be safely cloned, the user runs Get-ADDCCloningExcludedApplicationList cmdlet again with the -GenerateXML option in order to add it to the CustomDCCloneAllowList.xml file. For more information, see [Step 2: Run Get-ADDCCloningExcludedApplicationList cmdlet](https://docs.microsoft.com/powershell/module/addsadministration/get-addccloningexcludedapplicationlist).  
```

File: WindowsServerDocs/identity/ad-ds/reference/virtual-dc/Virtualized-Domain-Controller-Cloning-Test-Guidance-for-Application-Vendors.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ ms.technology: identity-adds
...
-The virtualized domain controller cloning process is described in detail in [Introduction to Active Directory Domain Services (AD DS) Virtualization (Level 100)](https://technet.microsoft.com/library/hh831734.aspx) and [Virtualized Domain Controller Technical Reference (Level 300)](https://technet.microsoft.com/library/jj574214.aspx). From an application vendor's perspective, these are some considerations to take into account when assessing the impact of cloning to your application:  
+The virtualized domain controller cloning process is described in detail in [Introduction to Active Directory Domain Services (AD DS) Virtualization (Level 100)](https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/introduction-to-active-directory-domain-services-ad-ds-virtualization-level-100) and [Virtualized Domain Controller Technical Reference (Level 300)](https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/deploy/virtual-dc/virtualized-domain-controller-technical-reference--level-300-). From an application vendor's perspective, these are some considerations to take into account when assessing the impact of cloning to your application:
```
Comment (ItzLevvie) [inline WindowsServerDocs/identity/ad-ds/reference/virtual-dc/Virtualized-Domain-Controller-Cloning-Test-Guidance-for-Application-Vendors.md]:
```suggestion
The virtualized domain controller cloning process is described in detail in [Introduction to Active Directory Domain Services (AD DS) Virtualization (Level 100)](https://docs.microsoft.com/windows-server/identity/ad-ds/introduction-to-active-directory-domain-services-ad-ds-virtualization-level-100) and [Virtualized Domain Controller Technical Reference (Level 300)](https://docs.microsoft.com/windows-server/identity/ad-ds/deploy/virtual-dc/virtualized-domain-controller-technical-reference--level-300-). From an application vendor's perspective, these are some considerations to take into account when assessing the impact of cloning to your application:  
```

**Comment by illfated** [comment]
I concur with the suggested changes. Even if the pages are only available in English (en-us) at this time, it is usually a good idea to use language agnostic links, just in case some of them are (or will become) available in other languages (except in the event that any of the links become broken again if the /en-us/ part is removed).

**Comment by eross-msft** [comment]
@ItzLevvie @mikeamoss  I believe since you added the change, you must use the **Commit suggestion** button for each of your suggestions. I can't merge anything until your suggestions are part of the actual PR. This must be done in the next 30 days or it will become necessary to close this PR.

**Comment by illfated** [comment]
As mentioned in the other PRs, I am willing to open a new PR to have the changes incorporated in a new PR to get the necessary changes added to the source.

**Comment by eross-msft** [comment]
@illfated  Closing this item. Please open a new PR so we can go ahead and accept these changes.

**Comment by illfated** [comment]
OK, so only this PR remains to resolve by opening a new PR with the same (and adjusted) links.

**Pull request closed without merging**